docs/basic-docs/atomicdex-api-20-dev/get_current_mtp.md

```diff
@@ -0,0 +1,52 @@
+# get\_current\_mtp
+
+The `get_current_mtp` method returns the Median Time Past (MTP) from electrum servers for UTXO coins. This information is useful for debugging, specifically in cases where an electrum server has been misconfigured.
+
+
+#### Arguments
+
+| Parameter | Type   | Description                        |
+| --------- | ------ | ---------------------------------- |
+| coin      | string | A compatible (UTXO) coin's ticker  |
+
+#### Response
+
+| Parameter  | Type   | Description      |
+| ---------- | ------ | ---------------- |
+| mtp        | integer| Unix timestamp   |
+
+#### :pushpin: Examples
+
+#### Command
+
+```bash
+curl --url "http://127.0.0.1:7783" --data "{
+    \"method\": \"get_current_mtp\",
+    \"userpass\": \"USERPASS\",
+    \"mmrpc\": \"2.0\",
+    \"params\": {
+        \"coin\": \"KMD\"
```

Comment:
@smk762 we should use `RICK` now instead of `KMD` please.
Omer state a good reason here [https://github.com/KomodoPlatform/developer-docs/pull/388#discussion_r931442459](https://github.com/KomodoPlatform/developer-docs/pull/388#discussion_r931442459)

Reply:
done :+1: